src/Website/Views/Shared/_Header.cshtml

```diff
@@ -16,7 +16,7 @@
 			<ul class="nav navbar-nav navbar-right">
 				<li><a href="#">Login</a></li>
 				<li><a href="#about">Register</a></li>
-				<li><a href="#contact">Contact Us</a></li>
+				<li><a href="ContactUs">Contact Us</a></li>
```

Comment:
To make sure this path works no matter where you are in the site, it should be relative.  (/contactus, most of the time).

MVC actually has a character that can be used to represent the installation folder structure (within IIS) to make sure it's relative within that:

href="~/ContactUs"

If we decide on that lowercase is the standard, we'll want to change this href too.